pkg/gofr/datasource/pubsub/mqtt/mqtt.go

```diff
@@ -17,8 +17,9 @@ import (
 )
 
 const (
-	publicBroker  = "broker.hivemq.com"
-	messageBuffer = 10
+	publicBroker        = "broker.hivemq.com"
+	messageBuffer       = 10
+	DefaultCloseTimeout = 250 * time.Millisecond
```

Comment:
Why not keep `DefaultCloseTimeout` as 0.
As we are not keeping any wait time at other places.
Moreover, why not to stop subscribing first and the disconnect?